connector/atlassian_crowd/atlassian_crowd.go

```diff
@@ -0,0 +1,447 @@
+// Package atlassian_crowd provides authentication strategies using Atlassian Crowd.
+package atlassian_crowd
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/dexidp/dex/connector"
+	"github.com/dexidp/dex/pkg/groups"
+	"github.com/dexidp/dex/pkg/log"
+)
+
+const (
+	// read operations of the /api/v4/user endpoint
+	scopeUser = "read_user"
+	// used to retrieve groups from /oauth/userinfo
+	// https://docs.gitlab.com/ee/integration/openid_connect_provider.html
```

Comment:
gitlab? 😉

Reply:
This plugin is based on gitlab plugin. Definitely code cleaning is required.

Reply:
Fixed.